docs/features/files.md

```diff
@@ -18,3 +18,19 @@ new GenericContainer(...)
                                       "/etc/redis.conf",
                                       BindMode.READ_ONLY)
 ```
+
+## Copying files to and from containers
```

Comment:
We should consider putting this section above the one that talks about the filesystem operations

Reply:
Shall we move this section to the top and add a callout box to the other APIs, that they are not supported for every Docker and CI setup?